routes/users.js

```diff
@@ -2,30 +2,22 @@ var express = require("express");
 var router = express.Router();
 var db = require("../db.js");
 
-router.route("/")
+router
+  .route("/")
   .get(function (req, res) {
-    res.statusCode = 200;
-    res.json({
-      "id": 1,
-      "display_name": "BobJoe",
-      "name": "Robert Joseph"
-    })
-  })
-  .put(function(req, res) {
-    res.statusCode = 200;
-    res.end();
+    // Get all users
   })
   .post(function (req, res) {
-    res.statusCode = 200;
-      res.json({
-      "id": 2,
-      "display_name": "Anon421235",
-      "name": "Michael Jackson"
-    })
-  })
-  .delete(function(req, res) {
-    res.statusCode = 200;
-    res.end();
+    let user = req.body;
+    db.users.insert(user).then((result) => {
+      if (result[0].affectedRows === 0) {    // If no rows are affected, send a 404.
+        res.statusCode = 404;
+        res.end();
+        return;
+      }
+       res.statusCode = 200;
+       return res.json.id;
```

Comment:
You'll have to use `result.insertId` like you did in books.js to return the id to the caller. 
`res.json({ "id": result.insertId });`